sourcemash/api/feeds.py

```diff
@@ -104,8 +104,8 @@ def post(self):
                 q = Queue('scrape', connection=REDIS_CONNECTION)
```

Comment:
Or, in create_worker, we check config to see if not in development config. This way create_worker returns None if in development. Then, instead of a try except block, we can just do ```if REDIS_CONNECTION```.